File: JavaScript/generator.js
const puppeteer = require('puppeteer-extra')
const StealthPlugin = require('puppeteer-extra-plugin-stealth')
const fs = require('fs');

var url = ("https://www.yeezysupply.com/")


puppeteer.use(StealthPlugin())
puppeteer.launch({ headless: true }).then(async browser => {
  const page = await browser.newPage()
  await page.goto(url)
  var cookies = await page.cookies();
  await browser.close()
  var arrayLength = cookies.length;
  for (var i = 0; i < arrayLength; i++) {
    if (cookies[i].name == "_abck") {
      console.log('Generated Akamai _abck Cookies ! ');
      fs.writeFile('cookies.txt', "\n" + cookies[i].value, { flag: 'a+' }, err => {
        if (err) {
          console.error(err)
          return
        }
      })
    }
}
})

File: Python/generator.py
from playwright.sync_api import sync_playwright
import json

url = ("https://www.yeezysupply.com/")


def main():
    with sync_playwright() as p:
        browser = p.webkit.launch()
        context = browser.new_context()
        page = context.new_page()
        page.goto(url)
        page.wait_for_timeout(3000)
        cookies = context.cookies()
        browser.close()
        data = json.dumps(cookies)
        new_data = json.loads(data)
        for abck_cookie in new_data:
            if abck_cookie['name'] == '_abck':
                with open("cookies.txt", 'a') as af:
                    af.write('\n' + abck_cookie['value'])
                    print(abck_cookie['value'])
        print('Generated Akamai _abck Cookies ! ')
main()
